docs/release-notes/10-4-0/README.md

```diff
@@ -84,7 +84,7 @@ View full 3D objects in OpenProject in IFC format. Select and display the model
 
 You can change between a 3D view or 2D view of the building model in OpenProject.
 
-![OpenPoject-BIM_2D](OpenPoject-BIM_2D.png)
+![OpenProject-BIM_2D](OpenPoject-BIM_2D.png)
```

Comment:
Note: I did not fix the png file name - that is a reference to an actual file, which has a typo in the name.

Let me know if you want the file renamed...

Reply:
Done